src/Features/Core/IntroduceVariable/AbstractIntroduceVariableService.cs

```diff
@@ -134,10 +117,39 @@ protected virtual bool BlockOverlapsHiddenPosition(SyntaxNode block, Cancellatio
                     }
                 }
             }
+            else if (state.InExpressionBodiedMemberContext)
+            {
+                await CreateConstantFieldActionsAsync(state, actions, cancellationToken).ConfigureAwait(false);
+                actions.Add(CreateAction(state, allOccurrences: false, isConstant: state.IsConstant, isLocal: true, isQueryLocal: false));
+                actions.Add(CreateAction(state, allOccurrences: true, isConstant: state.IsConstant, isLocal: true, isQueryLocal: false));
+            }
```

Comment:
Should we throw an else at the end here to catch any Contexts we might have missed?